client/src/components/NewsPanel.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div v-if="newsItems" class="container py-4" data-cy="news-items">
```

Comment:
![grafik](https://user-images.githubusercontent.com/42967368/146672685-e00f9339-b3b7-4716-ba38-e11d54c5392b.png)

* Also ich finde diesen leichten Unterstricht bei der Überschrift ansprechend, aber dann müsste der bitte auf allen Unterschriften, welche auf weißen Grund sind ergänzt werden
* Den Button "Alle News anzeigen" würde ich rausnehmen und einen Eintrag in der Navbar unter Listen ergänzen.
* Wenn keine News vorhaden sind, würde ich das Panel auf Home vollständig ausblenden

Reply:
- Probier mal aus, wie das aussieht;) Alle anderen Überschriften haben schon irgendwas drunter kleben, was dem Auge eine Linie führt. Da sah der Strick komisch aus. Hier war allerdings zu viel weiß.
- Hm, ich finde es logisch und praktisch ist mir aber auch nicht so wichtig dieser Kopf. In anderen Komponenten wie den Wertungen gibts den halt auch. (Da ist er mir aber wichtiger:P)
- Ist gefixt.

Reply:
Aber Nachrichten sind nunmal nicht ewig aktuell. Alte Nachrichten stellen nur für einen kleinen Teil der Userschaft eine Interessante Information dar. Wer das nachlesen möchte, findet ja auch über "Listen" dahin.

Deshalb "fott domet" :-D